services/core-web/common/actionCreators/noticeOfWorkActionCreator.js

```diff
@@ -26,6 +26,25 @@ export const fetchNoticeOfWorkApplication = (applicationGuid) => (dispatch) => {
     .finally(() => dispatch(hideLoading()));
 };
 
+export const fetchImportNoticeOfWorkSubmissionDocumentsJob = (applicationGuid) => (dispatch) => {
```

Comment:
would love to see a test for all the new redux actionCreators/reducer states/selectors